### PR TITLE
README: Added known issue for SecurityConfiguration/CredentialEncrypted

### PR DESCRIPTION
**Pull request by rakillen**
Added titles to each issue.
Corrected some general formatting.

### CODE REVIEW

File: KnownIssues.md [hunk 1]
```diff
@@ -1,63 +1,59 @@
...
-   (i.e. JRF, RestrictedJRF) will be targeted to the dynamic cluster by the WDT tools using the FMW WLST function
-   applyJRF.
+The following list are known issues. The issue may contain a workaround or an associated issue number.
 
-   If you have only non-JRF user server groups targeted to a dynamic cluster, you will see the following message:
+### Resource Assignment to Dynamic Clusters
 
-    WLSDPLY-12238: Unable to target non-JRF template server groups for domain type <your domain typedef> to dynamic cluster(s).
+**ISSUE**:
+The createDomain and updateDomain tools cannot target non-JRF product resources to dynamic clusters because WebLogic WLST will not assign resources associated with extension or custom template user server groups to dynamic clusters. Resources associated to JRF user server groups defined in the domain typedef (i.e. `JRF`, `RestrictedJRF`) will be targeted to the dynamic cluster by the WDT tools using the FMW WLST function `applyJRF`.
```
Comment (CarolynRountree) [inline KnownIssues.md:8]: This issue has been resolved. you can now target server groups to the dynamic cluster

File: KnownIssues.md [hunk 2]
```diff
@@ -1,63 +1,59 @@
...
 **ACTION**:
-   Run RCU before executing createDomain for JRF domains
+Run the WLS `rcu` command before executing createDomain for JRF domains
+
+### Object Names with Slashes
 
 **ISSUE**:
-   The discoverDomain or createDomain tool cannot handle object name with slashes and finished with  
-   warnings.  For example:
+The discoverDomain or createDomain tool cannot handle object name with slashes and finished with warnings. For example:
```
Comment (CarolynRountree) [inline KnownIssues.md:44]: This has been resolved

File: KnownIssues.md (hunk 1 above)
Comment (rakillen) [reply KnownIssues.md]: Removed this section

File: KnownIssues.md (hunk 2 above)
Comment (rakillen) [reply KnownIssues.md]: Removed this section

File: KnownIssues.md [hunk 3]
```diff
@@ -1,63 +1,31 @@
...
-   If you have only non-JRF user server groups targeted to a dynamic cluster, you will see the following message:
-
-    WLSDPLY-12238: Unable to target non-JRF template server groups for domain type <your domain typedef> to dynamic cluster(s).
-
-   You will not see this message if you have a mix of non-JRF and JRF user server groups targeted to the dynamic
-   cluster. WDT cannot detect if a user server group is associated to JRF, and therefore, whether the applyJRF will
-   target the user group resources to the dynamic cluster.
-
-**ACTION**:
+The following list are known issues. The issue may contain a workaround or an associated issue number.
```
Comment (rosemarymarano) [inline KnownIssues.md:3]: The following list are known issues. (awkward wording) ->Two choices: In the following list are known issues. or The following list contains known issues.

File: KnownIssues.md [hunk 4]
```diff
@@ -1,63 +1,31 @@
...
-      user server groups to the configured managed server, the resources are automatically targeted to the cluster
-      by WebLogic, which includes both the managed server and the dynamic servers.
+### Discover Domain Tool Severe Messages
 
 **ISSUE**:
-   The discoverDomain STDOUT contains many SEVERE messages about cd() and ls() when it is run against a 12.2.1 domain.
-   The discover tool navigates through the domain MBeans using wlst to determine which MBeans are present in a
-   domain. When it tests an MBean that is not present, the erroneous message is logged by Weblogic WLST.
-   There is no 12.2.1 PSU available to address this WLST problem. It is resolved in 12.2.1.1.
+The discoverDomain STDOUT contains many SEVERE messages about cd() and ls() when it is run against a 12.2.1 domain. The discover tool navigates through the domain MBeans using wlst to determine which MBeans are present in a domain. When it tests an MBean that is not present, the erroneous message is logged by Weblogic WLST. There is no 12.2.1 PSU available to address this WLST problem. It is resolved in 12.2.1.1.
```
Comment (rosemarymarano) [inline KnownIssues.md:8]: I would suggest code font for `discoverDomain `, `cd()`, and `ls()`.
12.2.1 -> 12.2.1.0
discover tool -> Discover Domain Tool
wlst  -> WLST
the erroneous message -> an error message 
12.2.1 PSU -> 12.2.1.0 PSU

File: KnownIssues.md [hunk 5]
```diff
@@ -1,63 +1,31 @@
...
 
 **ACTION**:
-   Ignore the following messages logged during discover of a 12.2.1 domain.
-
-   ####<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: cd() failed.>
-   ####<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: ls() failed.>
+Ignore the following messages logged during discover of a 12.2.1 domain.
```
Comment (rosemarymarano) [inline KnownIssues.md:11]: during discover of a 12.2.1 domain -> during discovery of a 12.2.1.0 domain

File: KnownIssues.md [hunk 6]
```diff
@@ -1,63 +1,31 @@
...
+The following list are known issues. The issue may contain a workaround or an associated issue number.
 
-   You must contact WebLogic support to assist you with a solution for this targeting dilemma. You can perform
-   the following action as a temporary workaround to the described issue.
-
-   1. Add a configured managed server to your dynamic cluster and re-run the createDomain or updateDomain tool.
-      The dynamic cluster becomes a "mixed" cluster once the managed server is added. When the WDT tool targets the
-      user server groups to the configured managed server, the resources are automatically targeted to the cluster
-      by WebLogic, which includes both the managed server and the dynamic servers.
+### Discover Domain Tool Severe Messages
```
Comment (rosemarymarano) [inline KnownIssues.md:5]: Severe  -> SEVERE

File: KnownIssues.md [hunk 7]
```diff
@@ -1,63 +1,31 @@
...
-
-   ####<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: cd() failed.>
-   ####<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: ls() failed.>
+Ignore the following messages logged during discover of a 12.2.1 domain.
+```
+<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: cd() failed.>
+<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: ls() failed.>
+```
 
+### Create Domain Tool with 11G JRF Domains
```
Comment (rosemarymarano) [inline KnownIssues.md:17]: 11G -> 11g

File: KnownIssues.md [hunk 8]
```diff
@@ -1,63 +1,31 @@
...
 
 **ISSUE**:
-   The createDomain tool cannot run rcu for 11g JRF domains. The tool will issue error messages in the log, and
-   terminate the create process.
+The Create Domain Tool cannot initialize RCU for 11g JRF domains. The tool will issue error messages in the log, and terminate the create process.
```
Comment (rosemarymarano) [inline KnownIssues.md:20]: log, -> log (no comma needed)

File: KnownIssues.md [hunk 9]
```diff
@@ -1,63 +1,31 @@
...
-**ISSUE**:
-   The discoverDomain or createDomain tool cannot handle object name with slashes and finished with  
-   warnings.  For example:
-
-    2. WLSDPLY-06140: Unable to cd to the expected path /SelfTuning/NO_NAME_0/WorkManager/wm/SOAWorkManager
-    constructed from location context model_folders = ['SelfTuning', 'WorkManager'],  'name_tokens' =
-    {'SELFTUNING': 'NO_NAME_0','DOMAIN': 'basesoa','WORKMANAGER': 'wm/SOAWorkManager'}; the current folder and
-    its sub-folders cannot be discovered : wlst.cd(/SelfTuning/NO_NAME_0/WorkManager/wm/SOAWorkManager) in
-    offline mode failed: com.oracle.cie.domain.script.ScriptException: No such element WorkManager named wm
+Run the WLS `rcu` command before executing createDomain for JRF domains
```
Comment (rosemarymarano) [inline KnownIssues.md:23]: createDomain -> `createDomain` (code font)

File: KnownIssues.md [hunk 10]
```diff
@@ -1,63 +1,31 @@
...
 
-   The reason for this warning is because of the slash(es) in the object name. In this case the object name is
-   wm/SOAWorkManager
+### Credential in Security Configuration
 
-**ACTION**:
-   Contact Oracle Support to obtain the patch for the bug number 25790276 for your weblogic version
-   before running the tool.
+**ISSUE**: For WLS versions before 14.1.1, there is a problem setting the `CredentialEncrypted` attribute in the `topology/SecurityConfiguration` folder. The value is not encrypted properly in the configuration, and the domain will fail to start with the error:
```
Comment (rosemarymarano) [inline KnownIssues.md:27]: before  -> prior to
configuration, ->configuration (no comma needed)

File: KnownIssues.md [hunk 11]
```diff
@@ -1,63 +1,31 @@
...
+```
+java.lang.IllegalArgumentException: In production mode, it's not allowed to set a clear text value to the property: CredentialEncrypted of SecurityConfigurationMBean
+```
+**ACTION**: Contact Oracle Support to obtain the patch for the bug number 30874677 for your WebLogic version before running the tool.
```
Comment (rosemarymarano) [inline KnownIssues.md:31]: for the bug number -> for bug number
your WebLogic version -> your WebLogic Server version

**Pull request merged**